CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Go to [legend](#legend---types-of-changes) for further information about the typ
 - _WD_Option: Support for `DetailErrors` option
 
 ### Changed
+- _WD_FrameList
+	- Refactored for better performance
+	- Improved frame support
+	- Improved logging
```

Comment:
@mlipok Please review this for content / accuracy. Also need to know if you wanted `_WD_FrameList` to utilize the new detailed error reporting.

Reply:
> Also need to know if you wanted `_WD_FrameList` to utilize the new detailed error reporting.

Yes

> Please review this for content / accuracy.

will look today

Reply:
> Also need to know if you wanted `_WD_FrameList` to utilize the new detailed error reporting.

Please take a look: 
[$_WD_DetailedErrors support](https://github.com/Danp2/au3WebDriver/pull/434/commits/fe5c7507ef27b20f895b5028082145a7fc840326)

Reply:
> > Also need to know if you wanted `_WD_FrameList` to utilize the new detailed error reporting.
> 
> Yes
> 
> > Please review this for content / accuracy.
> 
> will look today

I'm fine with this changes.

Reply:
See [minor tweaks](https://github.com/Danp2/au3WebDriver/pull/434/commits/7ca53a1d3f6056e18d25849be55ffef8c1926fb1)

Reply:
> > Also need to know if you wanted `_WD_FrameList` to utilize the new detailed error reporting.
> 
> Yes

There are additional places in this function where the value of $iErr is modified. I think some of them should also be updated. Please review to see if you agree.

Reply:
> See [minor tweaks](https://github.com/Danp2/au3WebDriver/pull/434/commits/7ca53a1d3f6056e18d25849be55ffef8c1926fb1)

It looks fine